src/DataAccessLayer/dalMain.cs

```diff
@@ -32,6 +32,10 @@ public CosmosDal(Secrets secrets, Config config)
                 throw new ArgumentNullException(nameof(secrets));
             }
 
+#pragma warning disable IDE0017 // Simplify object initialization.
```

Comment:
Is there a way you can move this suppression to an external file? I'm wondering if something like [this](https://github.com/retaildevcrews/helium-csharp/blob/main/src/app/GlobalSuppressions.cs) would work in this case.

Reply:
moves suppression to a separate file.